test/helpers.js

```diff
@@ -50,15 +50,29 @@ exports.createCompiler = function createCompiler(options = {}) {
   });
   compiler.outputFileSystem = new MemoryFileSystem();
   return compiler;
-};
+}
 
-exports.removeCWD = function removeCWD(str) {
+export function countPlugins({ _plugins }) {
```

Comment:
@Kovensky @michael-ciniawsky Was for the test I hadn't added at first to this pr. But, I have since added it: https://github.com/webpack-contrib/uglifyjs-webpack-plugin/pull/74/files#diff-0570d5214a7cbff246a7ad90d119f15c